setup.py

```diff
@@ -7,27 +7,57 @@
 #
 
 import re
+import typing as tp
 from pathlib import Path
 
 from setuptools import find_packages, setup
 
+
+# Requirements
+
 requirements = {}
 for name in ["main", "dev"]:
     requirements[name] = Path(f"requirements/{name}.txt").read_text().splitlines()
 
+
+## Version
+
 init_str = Path("submitit/__init__.py").read_text()
 match = re.search(r"^__version__ = \"(?P<version>[\w\.]+?)\"$", init_str, re.MULTILINE)
 assert match is not None, "Could not find version in submitit/__init__.py"
 version = match.group("version")
 
 
+
+## Description
+
+def _replace_relative_links(regex: tp.Match[str]) -> str:
+    """Converts relative links into links to master
+    so that links on Pypi long description are correct
+    """
+    string: str = regex.group()
+    link = regex.group("link")
+    name = regex.group("name")
+    if not link.startswith("http") and Path(link).exists():
+        githuburl = "github.com/facebookincubator/submitit/blob/master"
```

Comment:
I think you should replace `master` by `{version}` here.

Reply:
ok, did it
on nevergrad I decided against because I am pretty sure master (until I rename it to `main`, anyway^^) will exist while the tag not necessarily. Fine with me anyway